searx/settings.yml

```diff
@@ -2466,6 +2466,19 @@ engines:
     shortcut: pgo
     disabled: true
 
+  - name: kagi
+    engine: kagi
+    shortcut: kg
+    categories: [general, web]
+    disabled: true
+    timeout: 4.0
+    api_key: ""
+    about:
+      website: https://kagi.com
+      use_official_api: false
+      require_api_key: true
+      results: HTML
```

Comment:
This should be commented out by default since the api key is required. Apart from that the `about` section here is redundant, it's already set in `kagi.py`.